packages/tools/pie-wrappers-react/scripts/react-gen-wrapper.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs-extra';
+import { appendFile } from 'fs'
+
+// fetches custom-elements.json file from the root
+const loadJSON = (path) => JSON.parse(fs.readFileSync(new URL(path, import.meta.url)));
+
+const customElementsObject = loadJSON(`../custom-elements.json`);
+
+export let components = []
+
+const customElements = Object.entries(customElementsObject)
+
+// sort through customElements array and put all components into a separate array
+customElements.forEach(([key, value]) => {
+    if ( key === 'modules') {
+        return value.forEach(k => {
+            k.declarations.forEach((decl) => {
+                decl.customElement === true ? components.push(decl) : ''
+            })
+        })
+    }
+})
+
+/**
+ * function to fetch events from the component
+ *
+ * @param {*} component object from within components array
+ * @param {*} event empty array to be populated with events
+ */
+function getEvents(component, events) {
```

Comment:
If there's no other reason for passing `events` as an argument, I'd embed it's initialization inside the `getEvents` function, and return its value at the function end.

```
function getEvents(component) {
  const events = [];
  ...
  return events;
}

components.forEach(component => {
  const events = getEvents(component)
  ...
}
```